docs/index.md

```diff
@@ -35,17 +35,19 @@ Structure](#package-structure).
 TODO: Improve usage message.
 
 ```text
-Usage: prefab [OPTIONS] [PACKAGEPATHS]...
+Usage: prefab [OPTIONS] [PACKAGE_PATH]...
```

Comment:
the paragraph on L30 says you need at least one package path, so this should be `[OPTIONS] PACKAGE_PATH...`

Reply:
I'll see if I can send a patch to the arg parsing library, since this is just the output of `--help`. I've fixed this in the docs for now though.